File: leetcode_1598/solution.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        level = 0
        for log in logs:
            if (log == "../"):
                if level<0:
                    level+=1
            elif (log == "./"):
                level = level
            else:
                level -=1
        return -level

File: leetcode_1598/solution.c
int minOperations(char** logs, int logsSize) {
    int level = 0;
    for (int i=0;i<logsSize;i++)
    {
        if (strcmp(logs[i], "../") == 0)
        {
            if (level<0)
                level+=1;
        }
        else if (strcmp(logs[i], "./") == 0)
            level = level;
        else
            level -=1;
    }
    return (-level);
}
